virtual-setup/slack-procedures.md

```diff
@@ -1 +1,117 @@
+* How we can improve the module (anonymous, only instructors will see your answer)
+
+We will also ask you what you want to work on the next day on days _preceding_ consultation days.
+Your answers will be sent to course instructors only and are **not** anonymous.
+This allows instructors to group course participants based on their interests.
+
+Use the `Submit Response` button in a Polly message to enter your feedback:
+
+<img src = screenshots/slack-polly-response.png width = 500>
+
+### Using Slack calling during training
+
+If you need help during instruction or on consultation days and it is helpful to share your screen, a course instructor will initiate a [Slack call](https://slack.com/help/articles/216771908-Make-calls-in-Slack).
+Your instructor will ask you to [share your screen](https://slack.com/help/articles/115003501303-Share-your-screen-with-Slack-Calls) using the share your screen icon circled in yellow below.
+
+<img src = screenshots/slack-call-control.png width = 300>
+
+When you share your screen in Slack, please note that the **entire screen** will be displayed.
+You can not select a specific window to share.
+
+_Note: Where possible please utilize headphones throughout the training workshop to minimize audio feedback._
+
+### Using direct messages during training
+
+If you have a question that is _highly specific_ to your own data, you may [direct message](https://slack.com/help/articles/212281468-What-is-a-direct-message) a CCDL staff member. 
+
+First, use the new message button in the top right side corner of the Slack interface.
+
+<img src = screenshots/slack-compose-new-message.png width=300>
+
+You are then able to search for the CCDL instructors – `Chante Bethell`, `Candace Savonen`, `Josh Shapiro` or `Jaclyn N. Taroni` – and compose your message.
+
+Course instructors may direct you to the training-specific channel for more general questions or to another instructor where appropriate.
+
+## General Slack use
+
+In this section, we include a collection of tips that you may find helpful for using throughout training that were not covered above.
+
+#### Attaching a file or image
+
+You can [add a file or image to Slack](https://slack.com/help/articles/201330736-Add-files-to-Slack) by dragging a file to the Slack window or by using the paper clip icon next to the message field:
+
+<img src = screenshots/slack-add-a-file.png width = 500> 
+
+#### Adding code blocks to messages 
+
+Code and error messages are usually easier to read when you use code formatting in Slack.
+You can either select the code block icon below the message field:
+
+<img src = screenshots/slack-code-block-icon.png width = 500>
+
+Or you can type three backticks <code>```</code> into the message field and formatting will be applied:
+
+<img src = screenshots/slack-backticks.gif width = 500 >
+
+You are then ready to paste your code or error message into the grey code block.
```

Comment:
Should we add a line at the end about reaching out to us if they have questions or having trouble working with some of the features?